static/src/components/Play.js

```diff
@@ -46,6 +47,8 @@ export class Play extends React.Component {
             single: false,
             militaryDialog: false,
             militaryOnce: false,
+            value: 6,
```

Comment:
Can we please make the default 2 so we can use less space in production db when testing :)